src/init/Cargo.lock

```diff
@@ -154,6 +184,12 @@ version = "3.16.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "79296716171880943b8470b5f8d03aa55eb2e645a4874bdbb28adb49162e012c"
 
+[[package]]
+name = "byteorder"
+version = "1.5.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1fd0f2584146f6f2ef48085050886acf353beff7305ebd1ae69500e27c67f64b"
+
```

Comment:
Also not included. Had a quick look: big file but looks okay: https://github.com/BurntSushi/byteorder/blob/master/src/lib.rs